Use docker-entrypoint when running consul (#590)
hashicorp/consul-k8s: CHANGELOG.md

IMPROVEMENTS:
* Add flags `-log-level`, `-log-json` to all subcommands to control log level and json formatting. [[GH-523](https://github.com/hashicorp/consul-k8s/pull/523)]

> * Execute consul clients and servers using the docker entrypoint to keep consistent. [[GH-590](https://github.com/hashicorp/consul-k8s/pull/590)]

Reviewer: Nit: "Consul" instead of "consul" and "Docker" instead of "docker" to be consistent with previous Changelog items.

Author: Nice catch!

> * Execute consul clients and servers using the docker entrypoint to keep consistent. [[GH-590](https://github.com/hashicorp/consul-k8s/pull/590)]

Reviewer 2: ```suggestion
* Execute consul clients and servers using the docker entrypoint for consistency. [[GH-590](https://github.com/hashicorp/consul-k8s/pull/590)]
```

> * Execute consul clients and servers using the docker entrypoint to keep consistent. [[GH-590](https://github.com/hashicorp/consul-k8s/pull/590)]

Author: ```suggestion
* Execute Consul clients and servers using the Docker entrypoint to keep consistent. [[GH-590](https://github.com/hashicorp/consul-k8s/pull/590)]
```
